tools/scripts/generate_binary_build_matrix.py

```diff
@@ -37,6 +37,7 @@
 WIN_GPU_RUNNER="windows-2019-m60"
 WIN_CPU_RUNNER="windows-2019"
 MACOS_M1_RUNNER="macos-m1-12"
+MACOS_XL_RUNNER="macos-12-xl"
```

Comment:
Maybe we should just use the maco-12 machines for this one because we only have a concurrency of 50 for `macos-12-xl`
```suggestion
MACOS_XL_RUNNER="macos-12"
```

Reply:
updated.